kyuubi-ha/src/main/scala/org/apache/kyuubi/ha/HighAvailabilityConf.scala

```diff
@@ -158,4 +158,12 @@ object HighAvailabilityConf {
       .version("1.5.0")
       .stringConf
       .createOptional
+
+  val DISCOVERY_CLIENT_CLASS: ConfigEntry[String] =
+    buildConf("kyuubi.service.discovery.client.class")
```

Comment:
Shall we change to `kyuubi.ha.discovery.client.class`, the key should put under [`HA`](https://github.com/apache/incubator-kyuubi/blob/master/docs/deployment/settings.md#ha) in settings.md .